npm2deb/scripts.py

```diff
@@ -20,6 +20,12 @@ def main(argv=None):
     parser.add_argument(
         '-v', '--version', action='version', version='%(prog)s ' + _.VERSION)
 
+    parser.add_argument(
```

Comment:
This is basically trying to do the same thing for any other parameter passed. 
I'm not sure how
`npm2deb create --no-registry /path/to/tar/file`
is any different from
`npm2deb rdepends --no-registry /path/to/tar/file`

Maybe you should either modify the logic here to work with just create - or alternatively maybe add in support like:

`npm2deb create-noregistry <node-module>`

Reply:
I misunderstood the execution path of the code: I thought that it always went through start(), as opposed to only after a call to create.  Thus, I thought my patch would change _all_ the subcommands, instead of just npm2deb create. 

I don't want to do the second suggestion, though.  npm2deb create-noregistry would probably require a whole new subparser, as opposed to just the same one with a new option.  I'll shift --no-registry to the create subparser.

The reason it was originally in the main parser was because, currently, none of the subcommands have man pages: meaning that they are undocumented, and so nobody would know about my shiny new patch.  I looked into auto-generating the extra manpages, but I couldn't find a working method.